src/blog/templates/blog/blog_page.html

```diff
@@ -37,7 +37,18 @@ <h1 class="bold text-4xl mb-0 text-black">
                 {{ page.body|richtext }}
             </article>
 
-            {% include "blog/includes/comments.html" with page=page %}
+            <div class="flex flex-row flex-wrap p-1">
+                {% for item in page.gallery_images.all %}
+                    <div class="flex-col p-1" style="width: 320px">
+                        {% image item.image fill-320x240 %}
+                        <p>{{ item.caption }}</p>
+                    </div>
+                {% endfor %}    
+            </div>
```

Comment:
I think the design could use some more attention here. Let's have a look at this screenshot I just made:

![afbeelding](https://github.com/drikusroor/brazil-blog/assets/8208970/ed494e2d-1b1a-44ee-87a1-18b0f75419f7)

1. The gallery section does not align with the content of the post. I would use the same padding as the content above, i.e. `p-4` (or `px-4`).
2. The "gap" between the images is made by adding padding to the images. Both flexbox and grid already support the `gap` css-property. You can use this in tailwindcss with `gap-{number}` ([docs](https://tailwindcss.com/docs/gap)). As the outer padding of the elements in the container are `p-4`, you will probably want to use `gap-4` here too.
3. Depending on the size of the viewport, there is whitespace on the right. I would like to suggest using `grid` here instead of `flex` and have it display over the full width. Depending on the size of the viewport you can change the amount of columns (e.g. `grid sm:grid-cols-2 md:grid-cols-3`.

Combining all of these would result into something like this (I also took the liberty to increase the image resolution a bit):

```suggestion
            <div class="grid sm:grid-cols-2 md:grid-cols-3 p-4 gap-4">
                {% for item in page.gallery_images.all %}
                    <div>
                        {% image item.image fill-480x360 %}
                        <p>{{ item.caption }}</p>
                    </div>
                {% endfor %}    
            </div>
```

Which would look like this:

![afbeelding](https://github.com/drikusroor/brazil-blog/assets/8208970/5072f199-03bc-482f-9e56-8ef7b1390afa)

![afbeelding](https://github.com/drikusroor/brazil-blog/assets/8208970/e344418b-1e83-440f-8ff3-c3c917877592)

![afbeelding](https://github.com/drikusroor/brazil-blog/assets/8208970/c3b1e533-9166-4d99-968d-281ecbd619e5)